docs/packages/pkg/IlmBase.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    IlmBase
+
+.. index::
+  single: media ; IlmBase
+
+.. _pkg.IlmBase:
+
+IlmBase
+=======
+
+-  `Official <https://github.com/AcademySoftwareFoundation/openexr>`__
+-  `Hunterized <https://github.com/cpp-pm/openexr>`__
+-  `Example <https://github.com/cpp-pm/hunter/blob/master/examples/IlmBase/CMakeLists.txt>`__
+-  Added by `Harry Mallon <https://github.com/hjmallon>`__ (`pr-N <https://github.com/cpp-pm/hunter/pull/N>`__)
```

Comment:
Can you update this to #138? I've been forgetting to do this...

Reply:
Done